pkg/scheduler/framework/preemption/preemption.go

```diff
@@ -122,6 +123,8 @@ type Interface interface {
 	// Note that both `state` and `nodeInfo` are deep copied.
 	SelectVictimsOnNode(ctx context.Context, state *framework.CycleState,
 		pod *v1.Pod, nodeInfo *framework.NodeInfo, pdbs []*policy.PodDisruptionBudget) ([]*v1.Pod, int, *framework.Status)
+	// DeleteVictim deletes the given victim pod from API server for preemption.
```

Comment:
nit: may want a bit more description on it.

```suggestion
	// DeleteVictim deletes the given victim pod from API server for preemption. 
	// Evaluator uses this function to delete the decided victim instead of calling the Pod delete API directly so that the preemption plugins can implement a hook on victim deletion.
```

Feel free to change the wording as you'd like. (I'm not good at English.

Reply:
Your English is excellent. I changed the wording slightly but included most of your comment. Thanks